Makefile

```diff
@@ -338,6 +338,9 @@ install:
 	ln -sf services		$(DESTDIR)/srv/salt/ceph/stage/4
 	ln -sf removal		$(DESTDIR)/srv/salt/ceph/stage/5
 
+install: copy-files
+	chown -R salt /srv/pillar/ceph
```

Comment:
@jan--f don't you also want to add this line:

```
sed -i "s/_REPLACE_ME_/`hostname -f`/" /srv/pillar/ceph/master_minion.sls
```

to before the chown command?